tfjs-backend-webgpu/src/kernels/FromPixels.ts

```diff
@@ -62,15 +63,18 @@ export function fromPixels(args: {
       [pixels.width, pixels.height];
   const outputShape = [height, width, numChannels];
 
-  const useImport = env().getBool('WEBGPU_USE_IMPORT') && isVideo;
+  const importVideo =
+      env().getBool('WEBGPU_IMPORT_EXTERNAL_TEXTURE') && isVideo;
   const isVideoOrImage = isVideo || isImage;
   if (isImageBitmap || isCanvas || isVideoOrImage) {
-    let texture: GPUTexture|GPUExternalTexture;
     let textureInfo: TextureInfo;
 
-    if (useImport) {
-      const externalTextureDescriptor = {source: pixels as HTMLVideoElement};
-      texture = backend.device.importExternalTexture(externalTextureDescriptor);
+    if (importVideo) {
+      if (!texture || (texture as GPUExternalTexture).expired) {
```

Comment:
It may be a different HTMLVideoElement. So you can't use a global `texture` to check whether we need to import.

Reply:
+1, you should have a {key: HTMLVideoElement, value: GPUExternalTexture} map for management

Reply:
Fixed in new PR. Please review it again.